bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,935 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="souliss"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<bridge-type id="gateway">
+		<label>Gateway</label>
+		<description>Represents a Souliss Gateway.</description>
+
+		<config-description>
+			<parameter name="GATEWAY_IP_ADDRESS" type="text" required="true">
+				<label>IP or Host Name</label>
+				<description>Will be resolved by discovery if auto configured
+				</description>
+			</parameter>
+			<parameter name="GATEWAY_PORT_NUMBER" type="integer" required="true">
+				<label>Gateway Port</label>
+				<description>Default is 230 UDP.
+				</description>
+				<default>230</default>
+			</parameter>
+			<parameter name="PREFERRED_LOCAL_PORT_NUMBER" type="integer" required="true">
+				<label>Preferred Local Port Number</label>
+				<description>Default port is 0. It means that it is random.
+				</description>
+				<default>0</default>
+			</parameter>
+
+			<parameter name="PING_INTERVAL" type="integer" min="1" max="300">
+				<label>Ping interval</label>
+				<description>Interval in seconds to check for device presence.</description>
+				<default>30</default>
+			</parameter>
+			<parameter name="SUBSCRIBTION_INTERVAL" type="integer" min="1" max="60">
+				<label>Subscription interval</label>
+				<description>Interval in minutes to subcribe Souliss Gateway.
+				</description>
+				<default>2</default>
+			</parameter>
+			<parameter name="HEALTHY_INTERVAL" type="integer" min="1" max="60">
+				<label>Healthy interval</label>
+				<description>Interval in seconds to send nodes healthy.
+				</description>
+				<default>35</default>
+			</parameter>
+
+			<parameter name="SEND_INTERVAL" type="integer" min="5" max="5000">
+				<label>Send interval</label>
+				<description>Interval in milliseconds to get packet from binding queue and send it to Souliss. First packet is sent
+					immediately.
+				</description>
+				<default>1000</default>
+			</parameter>
+
+			<parameter name="TIMEOUT_TO_REQUEUE" type="integer" min="0" max="10000">
+				<label>Interval in milliseconds to requeue packet in queue if not yet executed</label>
```

Comment:
1. Label is too long. Keep them short as in 2-3 words in length. Labels get used to create items from with new UI.
2.  Label should have Capital for each word, unless it is a joining word like 'or'.

```suggestion
				<label>Requeue Interval</label>
```

Reply:
fixed